Portfolio/index.html

```diff
@@ -4,27 +4,33 @@
     <title>Portfolio</title>
     <meta charset="UTF-8" />
     <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <div top="0">
+  <body>
+    <div class="cont1">
+
       <a href="../index.html">Go Back</a>
-    </div>
+
     <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
     <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
     <nav>
+    <h2>Jahnavi</h2>
         <ul>
             <li>Home</li>
             <li>About</li>
             <li>Skills</li>
             <li>Contact</li>
         </ul>
     </nav>
-    <h1>Hello I'm</h1>
-    <h2>Jahnavi</h2>
-    <img src="https://png.pngtree.com/element_our/20190601/ourlarge/pngtree-desktop-computer-one-flat-style-work-girl-vector-image_1358068.jpg" alt="working"/>
+    <i class="fa fa-phone"></i> <br>
+    <button class="button contact"><b>Contact Me</b></button>
+    <h1>Jahnavi Sai Priya</h1>
+    <img src="https://png.pngtree.com/element_our/20190601/ourlarge/pngtree-desktop-computer-one-flat-style-work-girl-vector-image_1358068.jpg" alt="working" width="200" height="200"/>
+    <pre><h1>Front End

```

Comment:
Consider adding a class or id to the 'h4' element for easier styling and targeting in CSS.